### PR TITLE
Updated info about product key

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/319

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/business/upgrade-to-windows-pro-creators-update.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ description: "Learn how to upgrade Windows devices to Windows 10 Pro Creators Up
      - Create an installation media using the [Media Creation Tool](https://go.microsoft.com/fwlink/?LinkID=836960) - Select this option to create a Windows 10 Pro Creators Update installation media (USB flash drive or ISO file) to install Windows 10 on a PC that's different from the one you're using.
     
         a. Read the instructions on how to use the tool and create your installation media. 
+
+> [!Note]
+> If you have Windows devices running Windows 7 Pro, Windows 8 Pro, or Windows 8.1 Pro, your Microsoft 365 Business subscription entitles you to a Windows 10 upgrade - you don't require a Product Key.
```
Comment (nenonix) [inline microsoft-365/business/upgrade-to-windows-pro-creators-update.md:37]:
```suggestion
> If you have devices running Windows 7 Pro, Windows 8 Pro, or Windows 8.1 Pro, your Microsoft 365 Business subscription entitles you to a Windows 10 upgrade - you won't need a product key.
```

**Review by nenonix — CHANGES_REQUESTED** [review]
copy review

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy Review - Agree with Nicole.

**Pull request merged**